bot/exts/utils/_eval_helper.py

```diff
@@ -278,14 +279,14 @@ def format_hastebin_output(self, output: dict, result: str) -> Embed:
         is more than 1991 characters or 40 lines.
         """
         logger.info("Formatting hastebin output...")
-        if result.count("\n") > 40:
+        if result.count("\n") > self.max_lines:
             result = [
                 f"{i:02d} | {line}" for i, line in enumerate(result.split("\n"), 1)
             ]
             result = result[: self.max_lines]  # Limiting to only 11 lines
             program_output = "\n".join(result) + "\n... (truncated - too many lines)"
 
-        elif len(result) > 1991:
+        elif len(result) > 1000:
```

Comment:
```suggestion
        elif len(result) > self.max_output_length:
```
Something like this should be used. We don't want hard coded values.